### PR TITLE
Repository Cleanup Part 1

### PR DESCRIPTION
**Pull request by zoey-kaiser**
Contributes to #226  .

Checklist:
- [x] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [x] issue checkboxes are all addressed
- [x] manually checked my feature / not applicable
- [x] wrote tests / not applicable
- [x] attached screenshots / not applicable

### CODE REVIEW

**Comment by zoey-kaiser** [comment]
- [X] StoryBook Works
- [X] npm Run Dev Works

**Comment by zoey-kaiser** [comment]
```
    1: "[Vue warn]: Unknown custom element: <SurplusAuction> - did you register the component correctly? For recursive components, make sure to provide the \"name\" option.·
    found in·
    ---> <Anonymous>
           <Root>"
    2: "[Vue warn]: Unknown custom element: <format-market-value> - did you register the component correctly? For recursive components, make sure to provide the \"name\" option.·
    found in·
    ---> <Anonymous>
           <Root>"
```

Tests are currently failing for these reasons, can you help me make some sense of them @valiafetisov? I checked google but it seems to have to do with Vue not finding them so I double checked that I correctly imported them everywhere. I sadly did not find a solution this way.

File: frontend/components/auction/collateral/CollateralAuction.stories.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+storiesOf('Auction/Collateral/CollateralAuction', module)
+    .add('Default', () => ({
+        ...common,
+        template: `<CollateralAuction :auction="auction" auctionId="1" @bid="bid" />`,
+    }))
+    .add('Finished', () => ({
+        ...common,
+        template: `<SurplusAuction :auction="auction" auctionId="1" @bid="bid" error="This auction is finished" />`,
```
Comment (valiafetisov) [inline frontend/components/auction/collateral/CollateralAuction.stories.js:29]: You use `SurplusAuction` component here while it's not provided inside `components` nor it should be used here

File: frontend/components/common/formatters/FormatPercentage.stories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+};
+
+storiesOf('Common/Formatters/FormatPercentage', module).add('Default', () => ({
+    ...common,
+    data() {
+        return {
+            amount: new BigNumber(faker.datatype.number(1)),
+        };
+    },
+    template: '<format-market-value :value="amount"/>',
```
Comment (valiafetisov) [inline frontend/components/common/formatters/FormatPercentage.stories.js:17]: Same here

File: frontend/components/common/other/FeaturesView.vue [hunk 3]
```diff
@@ -12,7 +12,7 @@
 <script lang="ts">
 import Vue from 'vue';
 import LandingBlock from '~/components/layout/LandingBlock.vue';
-import FeatureList from '~/components/FeatureList.vue';
+import FeatureList from '~/components/common/other/FeatureList.vue';
```
Comment (valiafetisov) [inline frontend/components/common/other/FeaturesView.vue:15]: I think `FeatureList` is a part of the unified page. Can you please assemble all related components under a separate `unified` folder? (sorry if I missed this in the proposal)

File: frontend/components/wallet/WalletTable.vue [hunk 4]
```diff
@@ -64,13 +64,13 @@
...
-import FormatCurrency from '../utils/FormatCurrency.vue';
-import TimeTill from '../common/TimeTill.vue';
-import Loading from '../common/Loading.vue';
-import BaseButton from '../common/BaseButton.vue';
-import TextBlock from '../common/TextBlock.vue';
-import WalletBlock from '../transaction/WalletBlock.vue';
+import FormatAddress from '../common/formatters/FormatAddress.vue';
+import FormatCurrency from '../common/formatters/FormatCurrency.vue';
+import TimeTill from '../common/formatters/TimeTill.vue';
+import Loading from '../common/other/Loading.vue';
```
Comment (valiafetisov) [inline frontend/components/wallet/WalletTable.vue:70]: Please use absolute imports where possible (except only for stories, where you already fixed the opposite 😉)

File: frontend/components/panels/MKRCheckPanel.vue [hunk 5]
```diff
@@ -1,121 +0,0 @@
-<template>
```
Comment (valiafetisov) [inline frontend/components/panels/MKRCheckPanel.vue:1]: What is the new name of this panel? Github didn't recognise the move, I also couldn't find it 🙂

Comment (zoey-kaiser) [reply frontend/components/panels/MKRCheckPanel.vue:1]: It was left over from #343. Lukas and I were not sure if we wanted to display two components (MKR and allowance check) in one or two panels. 

That is why we merged both versions to check later. As we decided later on to separate them, this component was never used, but I forgot to delete it.

File: frontend/components/common/other/FeaturesView.vue (hunk 3 above)
Comment (zoey-kaiser) [reply frontend/components/common/other/FeaturesView.vue:15]: FeatureList is actually not a part of the unified page. It is part of `/features`. However I will still make the unified folder for the unified components 😉

File: frontend/components/wallet/WalletTable.vue (hunk 4 above)
Comment (zoey-kaiser) [reply frontend/components/wallet/WalletTable.vue]: Done! I went through every component and double checked that all were done via absolute imports!

**Review by valiafetisov — APPROVED** [review]

**Review by LukSteib — APPROVED** [review]
Tested all core functionality steps:
* swap execution
  * all auth steps
  * swap
  * withdraw collateral
* bid with dai on collateral auction
  * deposit dai to VAT 
  * bid with dai
  * withdraw collateral
* surplus auction
  * deposit MKR
  * bid
  * withdraw dai

---
There is one thing I figure (but unrelated to this PR hence still approving this). Just for documentation purposes:
I have a feeling that we do not sync properly wallet authorisation state between `surplus` and `collateral`. 
While auth to withdraw DAI from VAT via wallet popup was given and correctly indicated when on `/collateral` page, the same step is indicated as requiring authorisation when on `/surplus` page. -> Will create an issue for that

**Pull request merged**